src/Map/src/Marker.php

```diff
@@ -47,6 +49,7 @@ public function toArray(): array
             'title' => $this->title,
             'infoWindow' => $this->infoWindow?->toArray(),
             'extra' => $this->extra,
+            'identifier' => $this->identifier,
```

Comment:
```suggestion
            'id' => $this->id,
```

You must also update TypeScript types definition to reflect this change.